docs/source/release/v6.7.0/index.rst

```diff
@@ -31,8 +31,8 @@ reported any issues to us. Please keep on reporting any problems you
 have, or crashes that occur on our `forum`_.
 
 Installation packages can be found on our `download page`_
-which now links to sourceforge to mirror our download files around the world. You can also
-access the source code on `GitHub release page`_.
+which now links to the assets on our `GitHub release page`_, where you can also
```

Comment:
Git a page not found error on the github link but I'm assuming that's because the tag doesn't exist yet

The link was https://github.com/mantidproject/mantid/releases/tag/v6.7.0